docs/customizing.rst

```diff
@@ -213,6 +213,36 @@ Celery. The practical way with custom serialization may look like so::
 .. _Celery: http://www.celeryproject.org/
 
 
+Custom send_mail method
+-----------------------
+
+It's also possible to completely override the ``security.send_mail`` method to
+implement your own logic, like so:
+
+    from flask import Flask
+    from flask_security import Security, SQLAlchemyUserDatastore
+
+    security = Security()
+
+    def create_app(config):
+        """Initialize Flask instance."""
+
+        app = Flask(__name__)
+        app.config.from_object(config)
+
+        def custom_send_mail(subject, recipient, template, **context):
+            # implement your own logic here
+            pass
+
+        datastore = SQLAlchemyUserDatastore(db, User, Role)
+        security_ctx.send_mail = custom_send_mail
+
+        return app
+
+Note that the above ``security.send_mail_task`` override will be useless if you
```

Comment:
Have you consider using `.. note::` directive http://www.sphinx-doc.org/en/stable/markup/para.html#directive-note ?